.github/workflows/linux_meson.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       if: matrix.python-version == '3.10'
       run: |
         # Packages that are only needed for their annotations
-        python -m pip install -r mypy_requirements.txt
+        python -m pip install mypy==1.0.0 types-psutil typing_extensions
```

Comment:
I've moved the `mypy` requirements here.

Reply:
That looks fine to me, thanks. Resolving.